src/server/cluster/cluster_family.cc

```diff
@@ -612,7 +672,9 @@ void ClusterFamily::Register(CommandRegistry* registry) {
                   acl::kDflyCluster}
                    .HFUNC(DflyCluster)
             << CI{"READONLY", CO::READONLY, 1, 0, 0, acl::kReadOnly}.HFUNC(ReadOnly)
-            << CI{"READWRITE", CO::READONLY, 1, 0, 0, acl::kReadWrite}.HFUNC(ReadWrite);
+            << CI{"READWRITE", CO::READONLY, 1, 0, 0, acl::kReadWrite}.HFUNC(ReadWrite)
+            << CI{"MGRTCONF", CO::ADMIN | CO::LOADING, -3, 0, 0, acl::kMGRTConf}.HFUNC(
```

Comment:
why do we allow this when LOADING?

Reply:
I believe we will have several internal commands here, the first one you currently created and the next ones will be to start the migration on each shard, so I think this command should have subcommand
you can call this DFLYMIGRATE and the subcommand can be REQUEST for the one you just implemented